README.md

```diff
@@ -18,4 +20,24 @@ bot.on('media_group', (ctx) => {
   }
   return ctx.reply(`total: ${ctx.mediaGroup.length}`)
 })
-```
\ No newline at end of file
+```
+
+## Telegraf v4
+```js
+const { Telegraf } = require('telegraf')
+const mediaGroup = require('.')
+
+const bot = new Telegraf(process.env.BOT_TOKEN)
+
+bot.use(mediaGroup())
+
+bot.on('message', async (ctx) => {
+  // ctx.mediaGroup — an array of album messages (including the last one)
+  for (const message of ctx.mediaGroup) {
```

Comment:
Isn't it `undefined` if this message is not part of media group?

Reply:
I thought this was just an example, and the check could be omitted, but I found a better solution for version 4, which gives more compatibility and does not require checks.